src/NUnitEngine/nunit.engine.tests/Services/DefaultTestRunnerFactoryTests.cs

```diff
@@ -89,6 +89,9 @@ public void ServiceIsStarted()
         [TestCase("x.dll y.dll z.dll",     "Multiple", typeof(MultipleTestProcessRunner))]
         public void CorrectRunnerIsUsed(string files, string processModel, Type expectedType)
         {
+            if (files == "EngineTests.nunit")
+                files = Path.Combine(TestContext.CurrentContext.TestDirectory, files);
+
```

Comment:
It's a little hackish, but since it's the only file that really expected to exist I think it's OK.